src/lib/github/utils.ts

```diff
@@ -44,3 +44,7 @@ export function mergeFileRepoIssues(repoIssues: RepoIssue[][]): RepoIssue[] {
       Object.keys(groupedByIssueType).map(key => groupedByIssueType[key]);
   return flatten<RepoIssue>(issueGroupsList);
 }
+
+export function formatStringArray (strng: String): String[] {
+  return strng.replace(/\s/g, "").replace(/[\[\]-]+/g, "").replace(/\(.*\)/, "").split(":");
```

Comment:
Comment text has committer author, can it be part of the returned items. 
see https://github.com/freelyformd/github-todos/blob/master/src/lib/github/issuePresenter.ts#L23 

Can you also rename this function to `getIssueCommentDetails` and then change return type to 
```
{
  file: string;
  comment: string;
  assignee: string;
}
```
I also think it might be better to add this function  in `src/lib/github/getComments.ts` module as its not that general.